### PR TITLE
Add rpc

### PR DESCRIPTION
**Pull request by vrtnd**

### CODE REVIEW

File: src/components/Aggregator/rpcs.ts [hunk 1]
```diff
@@ -28,11 +28,15 @@ function createProvider(name: string, defaultRpc: string, chainId: number, rando
 
 export const rpcUrls = {
 	1: {
-		default: 'https://rpc.ankr.com/eth',
+		default: 'https://eth.llamarewqpc.com',
```
Comment (0xngmi) [inline src/components/Aggregator/rpcs.ts:31]: wrong url?

File: src/components/Aggregator/rpcs.ts [hunk 2]
```diff
@@ -28,11 +28,15 @@ function createProvider(name: string, defaultRpc: string, chainId: number, rando
...
+		ankr: 'https://rpc.ankr.com/eth',
 		pokt: 'https://eth-mainnet.gateway.pokt.network/v1/5f3453978e354ab992c4da79',
 		cloudflare: 'https://cloudflare-eth.com',
 		linkpool: 'https://main-light.eth.linkpool.io',
-		mycryptoapi: 'https://api.mycryptoapi.com/eth'
+		mycryptoapi: 'https://api.mycryptoapi.com/eth',
```
Comment (0xngmi) [inline src/components/Aggregator/rpcs.ts:36]: isnt this rpc dead?

File: src/components/Aggregator/rpcs.ts (hunk 1 above)
Comment (vrtnd) [reply src/components/Aggregator/rpcs.ts]: checked how it will fallback and forgot to remove 😿

File: src/components/Aggregator/rpcs.ts [hunk 3]
```diff
@@ -83,11 +94,17 @@ export const rpcUrls = {
...
+		omniatech: 'https://endpoints.omniatech.io/v1/op/mainnet/public',
+		optimism: 'https://mainnet.optimism.io'
 	},
 	42161: {
 		default: 'https://arbitrum-one.blastapi.io/cfee5a54-245d-411b-ba94-da15d5437e88',
-		second: 'https://1rpc.io/arb'
+		onerpc: 'https://1rpc.io/arb',
+		omniatech: 'https://endpoints.omniatech.io/v1/arbitrum/one/public',
+		arb: 'https://arb1.arbitrum.io/rpc',
+		alchemy: 'https://arb-mainnet.g.alchemy.com/v2/demo'
```
Comment (0xngmi) [inline src/components/Aggregator/rpcs.ts:107]: best to remove this down, commonly down

**Pull request merged**